.test-infra/jenkins/job_LoadTests_ParDo_Python.groovy

```diff
@@ -131,7 +131,8 @@ def addStreamingOptions(test) {
     // Use the new Dataflow runner, which offers improved efficiency of Dataflow jobs.
     // See https://cloud.google.com/dataflow/docs/guides/deploying-a-pipeline#dataflow-runner-v2
     // for more details.
-    experiments: 'use_runner_v2',
+    // TODO(BEAM-11779) remove shuffle_mode=appliance with runner v2 once issue is resolved.
+    experiments: 'use_runner_v2, shuffle_mode=appliance',
```

Comment:
this has no effect as it should be a separate experiment. should this change be reverted now in it's entirety?